README.md

```diff
@@ -144,7 +144,7 @@ Default: false
 #### `service`
 
 Type: string
-Description: Which service to use. Supported options: [`slack`, `rocketchat`]
+Description: Which service to use. Supported options: [`slack`, `rocketchat`, `teams`]
```

Comment:
can you document a bit how to configure this?

Reply:
Will do, although there isn't much to configure.
Create an incoming webhook for the designated Teams channel, copy the webhook URI and add this to the webhook.yaml:
```yaml
chatops:
  enabled: true
  service: teams
  server_uri: "<Teams Webhook URI>"
  ```

Reply:
Added documentation.